atlasdb-feign/src/main/java/com/palantir/atlasdb/http/AtlasDbFeignTargetFactory.java

```diff
@@ -71,7 +72,7 @@ private AtlasDbFeignTargetFactory() {
                 .encoder(encoder)
                 .decoder(decoder)
                 .errorDecoder(errorDecoder)
-                .retryer(new InterruptHonoringRetryer())
+                .retryer(Retryer.NEVER_RETRY)
```

Comment:
Note that there's a semantics change here, as we can now throw Feign exceptions as opposed to general `RuntimeException`. I don't think this is a problem though?

Reply:
That's a good point, but should be fine

Reply:
re @jeremyk-91 point, we were already doing that once we'd run out of attempts, or are you specifically talking about the interrupted exception